### ISSUE TITLE
Rename the Weights File

### ISSUE THREAD

**Issue by LinjianLi** [kentsommer/keras-inceptionV4#26]
In the released version 2.1, if `This release targets Keras-2. These weights work with either Theano or Tensorflow (in either dim ordering)`, isn't it better to rename the weights file as
```
inception-v4_weights_tf_th_dim_ordering_tf_th_kernels-for_keras_2
inception-v4_weights_tf_th_dim_ordering_tf_th_kernels_notop-for_keras_2
```
to make them more "newbie-friendly" ?

**Comment by kentsommer** [comment]
Closing this out after a long hiatus from Github.

Happy to merge any pull requests if there is a desire to improve things!
